src/Protector.php

```diff
@@ -493,20 +504,24 @@ protected function getConfigValueForKey(string $key, $default = null): ?string
      *
      * @return \Illuminate\Contracts\Routing\ResponseFactory|\Illuminate\Http\Response|null
      */
-    public function generateFileDownloadResponse(Request $request, string $connectionName = null)
+    public function generateFileDownloadResponse($user, string $connectionName = null, bool $disableTokenCheck = false)
     {
+        if (!is_a($user, config('auth.providers.users.model'))) {
```

Comment:
If you just want to make sure this is any type of authenticable Laravel User, why not just 
```suggestion
        if (!is_subclass_of($user, Illuminate\Foundation\Auth\User::class)) {
```